orchestra/admin_views.py

```diff
@@ -28,8 +28,7 @@ def project_details(request, project_id):
 
 
 def _prettify_project_details(project, tasks_data, steps):
-    project['start_datetime'] = datetime.strptime(project['start_datetime'],
-                                                  '%Y-%m-%dT%H:%M:%S.%fZ')
+    project['start_datetime'] = parse_datetime(project['start_datetime'])
```

Comment:
It's using the set timezone for the project—`start_datetime` gets stored as timezone-aware, and `parse_datetime` will read it out in the same way.